src/pybind/mgr/dashboard/frontend/src/app/ceph/performance-counter/performance-counter/performance-counter.component.html

```diff
@@ -1,6 +1,19 @@
+<nav aria-label="breadcrumb">
```

Comment:
Just out of curiosity, and a perhaps-naive comment (and understanding this is a backporting): instead of adding templates/logic for every specific component, why not using an Angular module for breadcrumbs? Like [angular2-crumbs](https://www.npmjs.com/package/angular2-crumbs) (just googled it, haven't worked with it)?

Reply:
> I agree that having a component for breadcrumb is an improvement, and deserves a dedicated issue.

Thanks for the suggestion - I filed https://tracker.ceph.com/issues/24781 for keeping track of this.

Reply:
Thanks @LenzGr

Reply:
Yeah, thanks @LenzGr